web-ui/router.ml

```diff
@@ -182,6 +195,19 @@ let github_routes github =
           List.hd (Astring.String.cuts ~sep:"/-/" (Dream.target request))
         in
         Dream.redirect request target);
+    Dream.post "/github/:org/:repo/commit/:hash/variant/:variant/cancel"
+      (fun request ->
+        Dream.form request >>= function
+        | `Ok _ ->
+            Controller.Github.cancel_step
+              ~org:(Dream.param request "org")
+              ~repo:(Dream.param request "repo")
+              ~hash:(Dream.param request "hash")
+              ~variant:(Dream.param request "variant")
+              request github
+        | _ ->
```

Comment:
Isn't there a more explicit error message we could use, or is that on purpose?

Reply:
Just a duplication of the build error message for the form.